### PR TITLE
Refactor location to address

### PR DESCRIPTION
**Pull request by NgZiXin**
Better fit use case

### CODE REVIEW

File: gradlew [hunk 1]
```diff
@@ -129,14 +129,14 @@ if [ -n "$JAVA_HOME" ] ; then
         die "ERROR: JAVA_HOME is set to an invalid directory: $JAVA_HOME
 
 Please set the JAVA_HOME variable in your environment to match the
-location of your Java installation."
+address of your Java installation."
```
Comment (leslieyip02) [inline gradlew:132]: I don't think this is correct.

File: gradlew.bat [hunk 2]
```diff
@@ -46,7 +46,7 @@ echo.
 echo ERROR: JAVA_HOME is not set and no 'java' command could be found in your PATH.
 echo.
 echo Please set the JAVA_HOME variable in your environment to match the
-echo location of your Java installation.
+echo address of your Java installation.
```
Comment (leslieyip02) [inline gradlew.bat:49]: I don't think this is correct.

File: src/main/java/seedu/address/storage/JsonUserPrefsStorage.java [hunk 3]
```diff
@@ -32,7 +32,7 @@ public Optional<UserPrefs> readUserPrefs() throws DataLoadingException {
 
     /**
      * Similar to {@link #readUserPrefs()}
-     * @param prefsFilePath location of the data. Cannot be null.
+     * @param prefsFilePath address of the data. Cannot be null.
```
Comment (leslieyip02) [inline src/main/java/seedu/address/storage/JsonUserPrefsStorage.java:35]: I don't think this is correct.

File: src/main/java/seedu/address/storage/buyer/JsonBuyerListStorage.java [hunk 4]
```diff
@@ -39,7 +39,7 @@ public Optional<ReadOnlyBuyerList> readBuyerList() throws DataLoadingException {
     /**
      * Similar to {@link #readBuyerList()}.
      *
-     * @param filePath location of the data. Cannot be null.
+     * @param filePath address of the data. Cannot be null.
```
Comment (leslieyip02) [inline src/main/java/seedu/address/storage/buyer/JsonBuyerListStorage.java:42]: I don't think this is correct.

File: src/main/java/seedu/address/storage/buyer/JsonBuyerListStorage.java [hunk 5]
```diff
@@ -67,7 +67,7 @@ public void saveBuyerList(ReadOnlyBuyerList buyerList) throws IOException {
     /**
      * Similar to {@link #saveBuyerList(ReadOnlyBuyerList)}.
      *
-     * @param filePath location of the data. Cannot be null.
+     * @param filePath address of the data. Cannot be null.
```
Comment (leslieyip02) [inline src/main/java/seedu/address/storage/buyer/JsonBuyerListStorage.java:70]: I don't think this is correct.

File: src/main/java/seedu/address/storage/meetup/JsonMeetUpListStorage.java [hunk 6]
```diff
@@ -39,7 +39,7 @@ public Optional<ReadOnlyMeetUpList> readMeetUpList() throws DataLoadingException
     /**
      * Similar to {@link #readMeetUpList()}.
      *
-     * @param filePath location of the data. Cannot be null.
+     * @param filePath address of the data. Cannot be null.
```
Comment (leslieyip02) [inline src/main/java/seedu/address/storage/meetup/JsonMeetUpListStorage.java:42]: I don't think this is correct.

File: src/main/java/seedu/address/storage/meetup/JsonMeetUpListStorage.java [hunk 7]
```diff
@@ -67,7 +67,7 @@ public void saveMeetUpList(ReadOnlyMeetUpList meetUpList) throws IOException {
     /**
      * Similar to {@link #saveMeetUpList(ReadOnlyMeetUpList)}.
      *
-     * @param filePath location of the data. Cannot be null.
+     * @param filePath address of the data. Cannot be null.
```
Comment (leslieyip02) [inline src/main/java/seedu/address/storage/meetup/JsonMeetUpListStorage.java:70]: I don't think this is correct.

File: src/main/java/seedu/address/ui/buyer/BuyerCard.java [hunk 8]
```diff
@@ -18,7 +18,7 @@ public class BuyerCard extends UiPart<Region> {
     private static final String FXML = "BuyerListCard.fxml";
 
     /**
-     * Note: Certain keywords such as "location" and "resources" are reserved keywords in JavaFX.
+     * Note: Certain keywords such as "address" and "resources" are reserved keywords in JavaFX.
```
Comment (leslieyip02) [inline src/main/java/seedu/address/ui/buyer/BuyerCard.java:21]: I don't think this is correct.

**Review by leslieyip02 — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/address/ui/property/PropertyCard.java [hunk 9]
```diff
@@ -17,7 +17,7 @@ public class PropertyCard extends UiPart<Region> {
     public final Property property;
 
     /**
-     * Note: Certain keywords such as "location" and "resources" are reserved keywords in JavaFX.
+     * Note: Certain keywords such as "address" and "resources" are reserved keywords in JavaFX.
```
Comment (CreditableX) [inline src/main/java/seedu/address/ui/property/PropertyCard.java:20]: Perhaps this is not correct...

File: gradlew (hunk 1 above)
Comment (NgZiXin) [reply gradlew:132]: yup, changing that thanks!

Comment (KennyLewi) [reply gradlew]: i think this is correct.

File: gradlew.bat (hunk 2 above)
Comment (KennyLewi) [reply gradlew.bat]: i think this is correct now.

File: src/main/java/seedu/address/storage/JsonUserPrefsStorage.java (hunk 3 above)
Comment (KennyLewi) [reply src/main/java/seedu/address/storage/JsonUserPrefsStorage.java]: i think this is correct now.

File: src/main/java/seedu/address/storage/buyer/JsonBuyerListStorage.java (hunk 4 above)
Comment (KennyLewi) [reply src/main/java/seedu/address/storage/buyer/JsonBuyerListStorage.java]: i think this is correct now.

File: src/main/java/seedu/address/storage/buyer/JsonBuyerListStorage.java (hunk 5 above)
Comment (KennyLewi) [reply src/main/java/seedu/address/storage/buyer/JsonBuyerListStorage.java]: i think this is correct now.

File: src/main/java/seedu/address/storage/meetup/JsonMeetUpListStorage.java (hunk 6 above)
Comment (KennyLewi) [reply src/main/java/seedu/address/storage/meetup/JsonMeetUpListStorage.java]: i think this is correct now.

File: src/main/java/seedu/address/storage/meetup/JsonMeetUpListStorage.java (hunk 7 above)
Comment (KennyLewi) [reply src/main/java/seedu/address/storage/meetup/JsonMeetUpListStorage.java]: i think this is correct now.

File: src/test/java/seedu/address/testutil/property/TypicalProperties.java [hunk 10]
```diff
@@ -23,35 +23,35 @@
...
 
     // Manually added - Property's details found in {@code CommandTestUtil}
     public static final Property AMY = new PropertyBuilder().withLandlordName(VALID_NAME_AMY).withPhone(VALID_PHONE_AMY)
             .withAskingPrice(VALID_ASKING_PRICE_AMY).withPropertyType(VALID_PROPERTY_TYPE_AMY)
-            .withLocation(VALID_LOCATION_AMY).build();
+            .withAddress(VALID_ADDRESS_AMY).build();
     public static final Property BOB = new PropertyBuilder().withLandlordName(VALID_NAME_BOB).withPhone(VALID_PHONE_BOB)
             .withAskingPrice(VALID_ASKING_PRICE_BOB).withPropertyType(VALID_PROPERTY_TYPE_BOB)
-            .withLocation(VALID_LOCATION_BOB).build();
+            .withAddress(VALID_ADDRESS_BOB).build();
```
Comment (KennyLewi) [inline src/test/java/seedu/address/testutil/property/TypicalProperties.java:54]: LGTM

**Review by KennyLewi — COMMENTED** [review]
LGTM

**Pull request merged**